core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -524,18 +527,27 @@ object ConsumerGroupCommand extends Logging {
     val describeOpt = parser.accepts("describe", DescribeDoc)
     val deleteOpt = parser.accepts("delete", DeleteDoc)
     val newConsumerOpt = parser.accepts("new-consumer", NewConsumerDoc)
+    val groupInitializationTimeoutMsOpt = parser.accepts("group-init-timeout", GroupInitializationTimeoutMsDoc)
```

Comment:
I wonder if this name is a bit low-level for users? Could it just be "timeout"? That would allow us to potentially reuse it other scenarios.